static/js/apps/explore_landing/topics.json

```diff
@@ -26,13 +26,64 @@
             "url": "oq=Health+equity+in+Santa+Clara+County&t=dc/topic/HealthEquity&p=geoId/06085"
           }
         ],
-        "specific": [],
-        "comparison": []
+        "specific": [
+          {
+            "title": "Number of poor hispanic women in Santa Clara County",
+            "url": "oq=Number+of+poor+hispanic+women+in+Santa+Clara+County&p=geoId/06085"
+          },
+          {
+            "title": "Counties with the highest unemployment rate in the US",
+            "url": "oq=Counties+with+the+highest+unemployment+rate+in+the+US&t=dc/topic/UnemploymentRate&p=country/USA&pt=County"
+          },
+          {
+            "title": "Counties with the highest rates of uninsured",
+            "url": "oq=Counties+with+the+highest+rates+of+uninsured&t=dc/topic/NoHealthInsurance&p=country/USA&pt=County"
+          },
+          {
+            "title": "Cities with the highest number of women PhDs",
+            "url": "oq=Cities+with+the+highest+number+of+women+PhDs&t=Count_Person_25OrMoreYears_EducationalAttainmentDoctorateDegree_Female&t=dc/topic/EducationalAttainmentByGenderAndLevel&p=country/USA&pt=City"
+          },
+          {
+            "title": "Countries with the lowest Gini index",
+            "url": "oq=Countries+with+the+lowest+Gini+index&t=dc/svpg/dc/topic/EconomicEquity_GiniIndex&p=Earth&pt=Country"
```

Comment:
Lets not link to dc/svpgs -- https://screenshot.googleplex.com/8YTkS3bZey6d285 ?

Reply:
Thanks! Switched this one to the SV `GiniIndex_EconomicActivity`